### ISSUE TITLE
Exported 3D Point Geometeries Get Flattened (3D -> 2D)

### ISSUE THREAD

**Issue by sonofwatt84** [evetion/GeoDataFrames.jl#59]
When exporting a GeoDataFrame that has 3D point geometries using GeoDataFrame.write(),  the exported 3D geometries get flattened to 2D.  This appears to be a geometry typing issue.

It can be corrected by altering the "lookup_type" dictionary in io.jl to associate PointTraits with wkbPoint25D geometries instead of wkbPoint geometries like the following:

```
const lookup_type = Dict{DataType,AG.OGRwkbGeometryType}(
    AG.GeoInterface.PointTrait => AG.wkbPoint25D,
    AG.GeoInterface.MultiPointTrait => AG.wkbMultiPoint,
    AG.GeoInterface.LineStringTrait => AG.wkbLineString,
    AG.GeoInterface.LinearRingTrait => AG.wkbMultiLineString,
    AG.GeoInterface.MultiLineStringTrait => AG.wkbMultiLineString,
    AG.GeoInterface.PolygonTrait => AG.wkbPolygon,
    AG.GeoInterface.MultiPolygonTrait => AG.wkbMultiPolygon,
)

```

There's probably a more elegant solution using GeoInterface, but this hack is useful if you only ever work in 3D points.

**Comment by evetion** [comment]
Thanks for the report! I assume you're only using xyz points (given that other geometries require a treatment like this too?).

**Comment by sonofwatt84** [comment]
Yup, I'm only using XYZ points, which is why the proposed hack only applies
changes there.  I know ArchGDAL seems to make a distinction between
geometries with 2D and 2.5D verticies, but I couldn't see how that
distinction can be made with a utility like Geointerface.geomtrait

**Comment by sonofwatt84** [comment]
Thanks for the quick fix!  I'm using 0.3.7 now, and it works great!
